spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
+  describe "GET games#new" do
+    it 'responds successfully with an HTTP 200 status code' do
+      get :new
+      expect(response).to be_success
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe "GET games#show" do
+    it 'responds successfully with an HTTP 200 status code' do
+      get :show
+      expect(response).to be_success
```

Comment:
For test readability I like to separate Data setup, the action and the expectations.

So for this case I'd do
```Ruby
get :show

expect(response).to be_success
expect(response).to have_http_status(200)
```

Super small detail, but makes it easier to read :)